### PR TITLE
(feature) Codespaces - Added initial support for codespaces.

### PR DESCRIPTION
**Pull request by ShreyasJejurkar**
This PR adds support for codespaces for the fast blazor repository.
With this, we can
1. Make an edit to the project code. 
2. build it and run it directly into the cloud and test changes. 

Along with this I have also added commonly performed tasks to `tasks.json`. Press CTRL+SHIFT+P and choose a run task and run any of the tasks directly from VSCode. 

Currently, the following tasks have been added. 
1. Build library - Builds the library
2. Run samples - Prompts to choose sample (Server/Client) and based on the selection the sample will be run.

### CODE REVIEW

**Review by awentzel — APPROVED** [review]

**Review by nicholasrice — APPROVED** [review]

**Review by EisenbergEffect — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
@vnbaaij This seems like a cool improvement to me. I'd like to get your review and approval on this before we decide to merge, since you are the most frequent contributor and the person most likely to be affected by this change on a more regular basis. Thanks!

**Review by vnbaaij — APPROVED** [review]
Yes, good improvement indeed. As far as I know, you still need to have write rights to the repo to be able to make changes/run stuff but if one forks the repo they can be up and running in a flash!

**Pull request merged**
